overlord/snapstate/backend/copydata.go

```diff
@@ -38,6 +39,14 @@ var (
 	mkdirAllChown = osutil.MkdirAllChown
 )
 
+func MockAllUsers(f func(options *dirs.SnapDirOptions) ([]*user.User, error)) func() {
+	old := allUsers
```

Comment:
nitpick: add a call to `osutil.MustBeTestBinary`

Reply:
Will do. Thanks!